### PR TITLE
Right content refinements

### PR DESCRIPTION
**Pull request by aaronshekey**
📢 This PR is based off #479 and not `main`. This PR goes further to implement some recent design explorations. It puts all the contents of the page in a card and specifies the colors from our in-progress design system for light and dark mode refinements.

Screenshot 2023-11-14 at 3 17 44 PM
Screenshot 2023-11-14 at 3 17 54 PM

### CODE REVIEW

File: src/css/custom.css
Comment (aaronshekey) [inline src/css/custom.css:280]: Ahhh of course. These cursed selectors will not work in a production environment. This could be tricky, after all. We may not have access to these elements.

File: src/css/custom.css
Comment (aaronshekey) [inline src/css/custom.css:284]: This compiles as `.sidebarViewport_uxQu` 🤔

Comment (blyonng) [reply src/css/custom.css:284]: div[class*="sidebarViewport"] should work (classname contains sidebarViewport)

**Comment by nijikokun** [comment]
@aaronshekey we should add additional padding to the bottom of the sidebar so that the bottom links are not covered by the browsers link preview interface:

Screenshot 2023-11-15 at 11 06 56 AM

Another thing we should try and figure out is how to make the sidebar scroll hide when not in usage like notion:

Screenshot 2023-11-15 at 11 07 41 AM

**Review by russorat — APPROVED** [review]
LGTM! once you fix the conflict, we can merge

**Pull request merged**
